Controllers/UserController.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Controllers;
+
+use Models\UserModel;
+
+class UserController implements IController
+{
+    private $model;
+    private $fc;
+
+    public function __construct()
+    {
+        $this->model = new UserModel();
+        $this->fc = FrontController::getInstance();
+    }
+
+    public function createUserAction()
+    {
+        $imgName = "";
+        if ($_FILES['imageUser']["error"] == UPLOAD_ERR_OK) {
+            $imgName = rand() . $_FILES['imageUser']['name'];
+            move_uploaded_file($_FILES['imageUser']['tmp_name'], 'Media/images/users/' . $imgName);
+        }
+        $name = $_POST["login"];
```

Comment:
I'd rather add one more instruction to check. Mb something went wrong and our post request wasn't sent?
add to your first **if** condition something like `... && !empty($_POST)...`